### PR TITLE
Enable rhel6 in both official builds and CI of corefx master

### PR DESCRIPTION
**Pull request by smile21prc**
Enable rhel6 in both official builds and CI of corefx master

### CODE REVIEW

**Comment by smile21prc** [comment]
@dotnet-bot test ci please

**Comment by smile21prc** [comment]
@dotnet-bot test ci microsoft/dotnet-buildtools-prereqs:centos-6-783abde-20171304101322 please

**Comment by mmitche** [comment]
test ci please

**Comment by smile21prc** [comment]
test ci please

File: buildpipeline/pipeline.json [hunk 1]
```diff
@@ -32,6 +32,23 @@
...
+          "Parameters": {
+            "PB_DockerTag": "centos-6-783abde-20171304101322",
+            "PB_BuildArguments": "-buildArch=x64 -Release -stripSymbols",
+            "PB_BuildTestsArguments": "-buildArch=x64 -Release -SkipTests -Outerloop -- /p:ArchiveTests=true /p:EnableDumpling=true",
+            "PB_SyncArguments": "-p -- /p:ArchGroup=x64",
+            "PB_TargetQueue": "RedHat.69.Amd64",
+            "PB_CreateHelixArguments": "/p:ArchGroup=x64 /p:ConfigurationGroup=Release /p:TestProduct=corefx /p:TimeoutInSeconds=1200 /p:TargetOS=Linux"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "Linux",
```
Comment (weshaggard) [inline buildpipeline/pipeline.json]: Change this to "RedHat6"

File: buildpipeline/pipeline.json [hunk 2]
```diff
@@ -343,6 +360,23 @@
...
+          "Parameters": {
+            "PB_DockerTag": "centos-6-783abde-20171304101322",
+            "PB_BuildArguments": "-buildArch=x64 -Debug",
+            "PB_BuildTestsArguments": "-buildArch=x64 -Debug -SkipTests -Outerloop -- /p:ArchiveTests=true /p:EnableDumpling=true",
+            "PB_SyncArguments": "-p -- /p:ArchGroup=x64",
+            "PB_TargetQueue": "RedHat.69.Amd64",
+            "PB_CreateHelixArguments": "/p:ArchGroup=x64 /p:ConfigurationGroup=Debug /p:TestProduct=corefx /p:TimeoutInSeconds=1200 /p:TargetOS=Linux"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "Linux",
```
Comment (weshaggard) [inline buildpipeline/pipeline.json]: Change this to "RedHat6"

**Comment by smile21prc** [comment]
Thanks Wes for the review, changed as suggested.

Will hold this merge until fully testing release/2.0.0 branch, and all commits to fix cli bootstrapp areported back to master.

**Comment by smile21prc** [comment]
Thanks @MattGal for the review.

File: buildpipeline/linux.groovy [hunk 3]
```diff
@@ -8,7 +8,16 @@
 
 def submittedHelixJson = null
 
-simpleDockerNode('microsoft/dotnet-buildtools-prereqs:rhel7_prereqs_2') {
+def dockerImageName
```
Comment (weshaggard) [inline buildpipeline/linux.groovy]: Lets undo the CI changes as they are going to require further changes (i.e. adjustments to the build commands) and it isn't clear to me whether or not we want to include them yet.

**Comment by smile21prc** [comment]
Thanks @weshaggard . Reverted changes to both .groovy files as suggested.

**Comment by weshaggard** [comment]
The coreclr bits merged now so once this is green go ahead and merge and test out (it is ok to wait until monday for that).

**Comment by smile21prc** [comment]
Got that, thanks @weshaggard .

**Pull request merged**